var/spack/repos/builtin/packages/py-execnet/package.py

```diff
@@ -14,6 +14,7 @@ class PyExecnet(PythonPackage):
     homepage = "https://codespeak.net/execnet"
     pypi = "execnet/execnet-1.7.1.tar.gz"
 
+    version("1.9.0", sha256="8f694f3ba9cc92cab508b152dcfe322153975c29bda272e2fd7f3f00f36e47c5")
     version("1.7.1", sha256="cacb9df31c9680ec5f95553976c4da484d407e85e41c83cb812aa014f0eddc50")
     version("1.4.1", sha256="f66dd4a7519725a1b7e14ad9ae7d3df8e09b2da88062386e08e941cafc0ef3e6")
 
```

Comment:
Python dep version is updated, apipkg is no longer used

Reply:
sigh, I'm still not used to `spack stage <pkg name>` "reusing" an old installed version :(